addon/templates/components/bs4/bs-form/element/errors.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if show}}
+  <div class={{feedbackClass}}>{{messages.firstObject}}</div>
+{{/if}}
```

Comment:
well personally i would prefer a div, imagine you have 2 error rules.
span will place the elements in a single line until it has to break
div are displayed as a block so they will display on 2 lines

i'm guessing that's why they changed it